GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
+
+The Operator Framework Community may adopt or amend its Governing
+Documents according to the following process:
+
+1. Propose a change by opening an issue in the community, and if there
+   seems to be general agreement, open a pull request to this document.
+2. The change should be announced via the mailing list along with an invitation for discussion on the Pull Request.
+3. A minimum of 7 days must pass between the announcement and the vote,
+   and the vote must remain open for a minimum of 7 days. The Steering
+   Committee may extend both periods.
+
+For a vote to pass, all of the following must occur:
+* A quorum of 3 Steering Committee members is required.
+* A 2/3 majority of the Steering Committee quorum must approve.
+
+## Code of Conduct Committee
+
+In order to review and enforce the Code of Conduct, the Steering
+Committee selects 3 people to be on the Code of Conduct Committee.
+These individuals will be chosen based on their community management and
+code of conduct experience, with diverse representation across the
+committee, including employer, gender, race, background, and region of
+the world. To avoid fatigue, the Steering Committee will replace at
+least 2 members of the CoC Committee each year. Members of
+the committee do not need to be members of Operator Framework if they
+have sufficient other expertise.
+
+The CoC Committee will receive reports of conduct violations confidentially,
+and will discuss them in closed meetings. If a report is determined to be a
+violation, they will recommend action on it appropriate to the scale, nature,
+and context of the violation, from requiring an apology, up to expulsion of an
+individual from the project. In the event that a contributor is to be demoted
+or expelled, the CoC Committee will forward this recommendation to the Steering
+Committee who will ratify it in a closed meeting. Should a member of the
+Steering Committee be the offender or the reporter, this recommendation will
+instead be forwarded to the CNCF staff for final arbitration.
+
+## Adding New Subprojects
+
+During the monthly Steering meeting, any project member may recommend projects
+to become part of Operator Framework. These projects should have the following
+characteristics:
+
+* Have a mission that aligns with the Operator Framework goals
+* Are appropriately licensed
+* Governance structure is fully documented in a GOVERNANCE file
+* Governance rules are modifiable by project leaders through a defined process.
+* Embrace the Operator Framework and CNCF values
+* Are under active development
+* Consist of high quality code and designs
+* Have at least 3 Approvers (Experimental Subprojects must have at least 1)
+
+Before submitting an application to the Steering Committee, the applying project
+must hold an internal vote of all major contributors to join
+Operator Framework. The Steering Committee will then review and vote on the
+application. A quorum of 3 Steering Committee members must vote, and
```

Comment:
Rather than having quorums, my suggestion is that you simply require all SC votes to be as a % of all existing Steering seats.  That is, if there's 7 commitee members then 4 are required for any majority vote and 5 for any 2/3 vote, regardless of how that vote is taken.  Quorums, especially minority quorums, must have rules around notice periods etc. that can be annoying to administer.